Readme.md

```diff
@@ -76,6 +76,7 @@ These operators are much like the RxSwift & RxCocoa core operators, but provide
 * [count](#count)
 * [partition](#partition)
 * [bufferWithTrigger](#bufferWithTrigger)
+* [Observable.merge(with:)](#mergewith)
```

Comment:
```suggestion
* [Observable.merge(with:)](#mergeWith) 
```

Did you check the playground link works at all?

Reply:
Yes, it works without that change